syntax = "proto3";

package message;

service VarifyService {
  rpc GetVarifyCode (GetVarifyRequest) returns (GetVarifyResponse) {}
}

message GetVarifyRequest {
  string email = 1;
}

message GetVarifyResponse {
  int32 error = 1;
  string email = 2;
  string code = 3;
}

message GetChatServerReq {
  int32 uid = 1;
}

message GetChatServerRsp {
  int32 error = 1;
  string host = 2;
  string port = 3;
  string token = 4;
}

service StatusService {
	rpc GetChatServer (GetChatServerReq) returns (GetChatServerRsp) {}
}
